mav_msgs/include/mav_msgs/default_topics.h

```diff
@@ -52,6 +52,7 @@ static constexpr char POSITION[] = "position";
 // Simulation-specific topic names.
 static constexpr char WRENCH[] = "wrench";
 static constexpr char WIND[] = "wind";
+static constexpr char WIND_SPEED[] = "wind_speed";
```

Comment:
oops sorry, just read the description now :)
Maybe we should rename wind to external force? Otherwise this is super confusing.